### ISSUE TITLE
Working with a non doorbell camera?

### ISSUE THREAD

**Issue by markcodes** [WouterGritter/ring-rtsp-stream#2]
Does this work with a non-doorbell camera?

**Comment by WouterGritter** [comment]
No, but you can modify it to do so (although a bash script would probably be just fine if your camera has an rtmp/rtsp stream).
This piece of code uses the `ring-client-api` package to generate an rtsp stream.
